src/styles/blocks/clock.scss

```
@@ -0,0 +1,85 @@
+.clock {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $clock-size;
+  height: $clock-size;
+  margin: auto;
+  border: 5px solid gold;
+  border-radius: 50%;
+  background-color: black;
+
+  &:after {
+    content: "";
+    position: absolute;
+    left: 50%;
+    top: 50%;
+
+    width: 2%;
+    height: 2%;
+
+    border-radius: 50%;
+    background-color: goldenrod;
+    transform: translate(-50%, -50%);
+  }
+
+  &__minutes:after {
+    content: "";
+    position: absolute;
+    left: 49%;
+    top: 20%;
+
+    width: 2%;
+    height: 30%;
+    border-radius: $border-radius;
+
+    background-color: whitesmoke;
+    transform-origin: 50% 100%;
+    animation: rotate 360s infinite linear;
+  }
+
+  &__seconds:after {
+    content: "";
+    position: absolute;
+    left: 49%;
+    top: 5%;
+
+    width: 2%;
+    height: 45%;
+    border-radius: $border-radius;
+
+    background-color: goldenrod;
+    transform-origin: 50% 100%;
+    animation: rotate 6s infinite linear;
+  }
+
+  &__numbers {
+    display: flex;
+    flex-direction: column;
+    color: gold;
+    font-size: 50px;
+    height: 100%;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+  }
+
+  &__row {
+    padding: 10px;
+
+    &-2 {
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      flex-direction: row;
+    }
```

```suggestion
  &__row-2 {
    padding: 10px;
    width: 100%;
    display: flex;
    justify-content: space-between;
    flex-direction: row;
    }
```

it doesn't look like `2` is the modifier or you have any class like `row-3...infinite.